src/pages/Home.jsx

```diff
@@ -6,15 +6,22 @@ import Main from "../components/Main/Main";
 import { useTranslate } from "react-translate";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
-  faGlobe,
-  faBoxes,
-  faBolt,
-  faParachuteBox,
+  faGlobe as box1Icon,
```

Comment:
im not sure about this, and it is definitely a matter of opinion, but in this case I think it makes sense to give them aliases

Reply:
Absolutely. Thanks for pointing it out.